releases/1.23/getstarted.md

```diff
@@ -0,0 +1,14 @@
+
+# Charmed Distribution of Kubernetes
```

Comment:
Where did this file get sourced from? There is a `{details}` thing that makes it look like a template from somewhere. Anyhow, do we really need it?